core/src/main/scripts/importer/validateData.py

```diff
@@ -2461,7 +2461,10 @@ def checkLine(self, data):
                 pass
             elif data_type == 'NUMBER':
                 if not self.checkFloat(value):
-                    self.logger.error(
+                	if (value[0] in ('>','<')) and value[1:].isdigit():
+                		pass
+                	else:
+                		self.logger.error(
```

Comment:
@zhx828 @kalletlak I think this is prolly ok to merge, but maybe we should create a new issue to indicate that that there should be a NUMBER_RAGE data_type or something along those lines. What do you think?

also how is this handled on the study view for barcharts like e.g.:

![Screen Shot 2020-02-19 at 6 11 13 PM](https://user-images.githubusercontent.com/1334004/74885402-70497d00-5343-11ea-8173-ea77b8e22fd6.png)

Reply:
so @rmadupuri validated that this works by importing some studies like this to triage. But yeah still curious what y'all think @zhx828 @kalletlak

Reply:
@inodb this kind of data has been in the portal for quite while now. SV handles it by treating it as the max/min value then group the rest of data together. I'm ok with NUMBER_RANGE, but no sure how frequent it will be used though.